games-roguelike/hengband/hengband-1.6.2-r1.ebuild

```diff
@@ -44,21 +44,17 @@ src_configure() {
 	local myconf
 	use l10n_ja || myconf="--disable-japanese"
 
-	egamesconf \
-		--with-setgid=${GAMES_GROUP} \
+	econf \
```

Comment:
Did egamesconf do any other magic for us?

Note that the [documentation](https://devmanual.gentoo.org/eclass-reference/games.eclass/index.html) says:
```
egamesconf [<args>...]
Games equivalent to 'econf' for autotools based build systems. It passes the necessary games specific directories automatically.
```

Looking at the [source](https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/games.eclass#n131), we can see it does a lot:
```
       [...]

	econf \
		--prefix="${GAMES_PREFIX}" \
		--libdir="$(games_get_libdir)" \
		--datadir="${GAMES_DATADIR}" \
		--sysconfdir="${GAMES_SYSCONFDIR}" \
		--localstatedir="${GAMES_STATEDIR}" \
		${_gamesconf} \
		"$@"
```


So, TL;DR: don't rush! :)

Reply:
I actually had checked this, but based on what i understand about the whole "getting rid of games.eclass" thing, game packages are supposed to be just normal packages now.

I checked, and autotool.eclass seems to set some sane values for these, but to your point there's probably something games.eclass was doing for me before that's no longer being done.

So I'll take a closer look and see what I find :)